README.md

```diff
@@ -157,23 +155,45 @@ Search parameters
 #### Get info on user
 
 ```sh
-curl -X 'GET' \
-  'https://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66' \
-  -H 'accept: application/ld+json'
+curl "http://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66"
 ```
 
 Here `ffdb7559-2ad3-4662-9fd4-d69849939b66` should be a `bruger` identifier.
 
 #### Get funktioner
 
 ```sh
-curl -X 'GET' \
-  'https://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66/funktioner' \
-  -H 'accept: application/ld+json'
+curl "http://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66/funktioner"
 ```
 
 Here `ffdb7559-2ad3-4662-9fd4-d69849939b66` should be a `bruger` identifier.
 
+### Using the API from another docker compose setup
+
+To use the API you must use the `serviceplatformen_organisation_api_app`
+network.
+
+```sh
+networks:
+  organisation_api:
+    external: true
+    name: serviceplatformen_organisation_api_app
+
+services:
+  phpfpm:
+    networks:
+      - organisation_api
+    ...
+```
+
+The `nginx` container has been overridden to have the container name
+`organisation_api`. Calls from other projects already having a `nginx`
+container should use `organisation_api` to make calls to the API, i.e.
```

Comment:
There's no need to mention that the `nginx` service has another name. Just tell that the API is now available on the address `http://organisation_api:8080`.

Reply:
Fixed in 1261909f22ec9ca3aa56801f3b440bfb1f3948f0.